gensim/corpora/lowcorpus.py

```diff
@@ -166,7 +166,7 @@ def save_corpus(fname, corpus, id2word=None, metadata=False):
 
         if truncated:
             logger.warning("List-of-words format can only save vectors with "
-                            "integer elements; %i float entries were truncated to integer value" %
+                            "integer elements; %i float entries were truncated to integer value" 
```

Comment:
the '%' sign should stay there for the substitution to work. Why was it removed?